tmva/tmva/inc/TMVA/DNN/Adam.h

```diff
@@ -70,7 +70,7 @@ class TAdam : public VOptimizer<Architecture_t, Layer_t, DeepNet_t> {
 public:
    /*! Constructor. */
    TAdam(DeepNet_t &deepNet, Scalar_t learningRate = 0.001, Scalar_t beta1 = 0.9, Scalar_t beta2 = 0.999,
-         Scalar_t epsilon = 1e-8);
+         Scalar_t epsilon = 1e-7);
```

Comment:
You chang ehere a default for Adam, this was intended?

Reply:
I have put same default as used in Tensorflow, 
see https://www.tensorflow.org/api_docs/python/tf/keras/optimizers/Adam
Since we are using the same convention. Epsilon is actually epsilon_hat defined in formula before paragraph 2.1 in Adam paper, https://arxiv.org/pdf/1412.6980.pdf

Reply:
Thank you for the review. 
I will remove the commented line in one of the mext commits